src/index.ts

```diff
@@ -130,10 +143,48 @@ export function getInstance(): EppoClient {
   return clientInstance;
 }
 
-function newEventDispatcher(sdkKey: string) {
-  const eventQueue = new FileBackedNamedEventQueue<Event>('events');
+function newEventDispatcher(
+  sdkKey: string,
+  config: IClientConfig['eventIngestionConfig'] = {},
+): EventDispatcher {
+  const {
+    batchSize = 1_000,
+    deliveryIntervalMs = 10_000,
+    retryIntervalMs = 5_000,
+    maxRetryDelayMs = 30_000,
+    maxRetries = 3,
+    maxQueueSize = 10_000,
+    disabled = false,
```

Comment:
Defaults match up with comments in `IClientConfig` 👍  Super minor nit, but would be nice for readability if these were in the same order